requirements/dev.txt

```diff
@@ -289,6 +280,10 @@ rfc3986==2.0.0
     # via
     #   -r requirements/quality.txt
     #   twine
+secretstorage==3.3.3
+    # via
+    #   -r requirements/quality.txt
+    #   keyring
```

Comment:
This chain of "added" items is just an artifact of some people running `make upgrade` on Linux and others on Mac -- `keyring` has a conditional dependency that varies by OS. I don't think it affects anything, though, since it just relates to a PyPI publishing mechanism that we don't even seem to use.